catalog/app/components/Markdown/Markdown.tsx

```diff
@@ -219,6 +246,8 @@ export const getRenderer = memoize(({ images, processImg, processLink }) => {
   md.use(imageHandler({ disable: !images, process: processImg }))
```

Comment:
since we're handling all images and links in the purify's hooks, we can (and should) remove this logic from markdown handlers so we're not doing it twice

Reply:
This makes sense, but I have some resistance, because first we do "right" handling using some AST, and then "hacky" handling using HTML elements mutation

Reply:
well, yeah, but
1. we do that "hacky" thing anyways
2. the "right" way doesn't handle html insertions, which are a supported part of the markdown standard, so it's incomplete

also, remarkable's AST thing looks like a joke TBH with all these string manipulations and lists of tokens...